Formula/armor.rb

```diff
@@ -0,0 +1,23 @@
+class Armor < Formula
+  desc "Simple HTTP server, supports HTTP/2 and auto TLS"
+  homepage "https://github.com/labstack/armor"
+  url "https://github.com/labstack/armor/archive/v0.1.1.tar.gz"
+  sha256 "1772c99dd16cb4cc05c27f34382c33c5ff60aa0f4703ad25d7d6127f2d0b745d"
+  head "https://github.com/labstack/armor.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    armorpath = buildpath/"src/github.com/labstack/armor"
+    armorpath.install buildpath.children
+
+    cd armorpath do
+      system "go", "build", "-o", bin/"armor", "cmd/armor/main.go"
+    end
+  end
+
+  test do
+    system "armor", "-v"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!